golang/lib/networks/network_context.go

```diff
@@ -473,3 +456,16 @@ func (networkCtx *NetworkContext) GetLambdas() (map[modules.LambdaID]bool, error
 
 	return lambdaIDs, nil
 }
+
+// ====================================================================================================
+// 									   Private helper methods
+// ====================================================================================================
+func (networkCtx *NetworkContext) getSharedDirectory(relativeServiceDirpath string, kurtosisEnclaveDataVolMountpointOnServiceContainer string) *services.SharedDirectory {
+
+	absFilepathOnThisContainer := networkCtx.enclaveDataVolMountpoint + "/" + relativeServiceDirpath
```

Comment:
Let's use `path.Join` just to make sure we're doing things as agnostic as possible